pkg/api/openshiftcluster_example.go

```diff
@@ -0,0 +1,9 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func exampleOpenShiftCluster() *OpenShiftCluster {
```

Comment:
Examples are out of scope of this PR, but I had to add few because existing validator tests depend on them: I didn't want to change the code where it's not necessary.